src/code42cli/cmds/users.py

```diff
@@ -60,6 +61,50 @@ def list_users(state, org_uid, role_name, active, inactive, format):
         formatter.echo_formatted_dataframe(df)
 
 
+@users.command()
+@click.argument("username")
```

Comment:
When commands have multiple arguments, we elect to use required options instead.
This is because then the order does not matter, like I don't have to use `--help` to see if the username goes first or the role name.

Reply:
Great, I added them as functions with flexible help text, assuming that they would be re-used in further expansions of the `users` command.